cli/tools/init/mod.rs

```diff
@@ -14,13 +14,19 @@ fn create_file(
   filename: &str,
   content: &str,
 ) -> Result<(), AnyError> {
-  let mut file = std::fs::OpenOptions::new()
-    .write(true)
-    .create_new(true)
-    .open(dir.join(filename))
-    .with_context(|| format!("Failed to create {filename} file"))?;
-  file.write_all(content.as_bytes())?;
-  Ok(())
+  let path = dir.join(filename);
+  if path.exists() {
+    info!("ℹ️ Skipped creating {filename} as already exists");
```

Comment:
Should it say `as it already exists`? We might want to put into a gray don't as well?

Reply:
Thanks, good catch. Pushed fixes in d8150afdffdca01c52ccc194674a3cf59b941fdb and  bfd5de73ab46ad76743f9c8ccba9b8ebab8b40d2.